src/main.py

```diff
@@ -62,9 +62,9 @@ def jsonify(weeks: Dict[int, Week], directory: str, canteen: Canteen, combine_di
     combined_df_name = "combined"
 
     # create directory for combined output
-    json_dir = f"{str(directory)}/{combined_df_name}"
+    json_dir = f"{directory}/{combined_df_name}"
     if not os.path.exists(json_dir):
-        os.makedirs(f"{str(directory)}/{combined_df_name}")
+        os.makedirs(json_dir)
```

Comment:
FYI (in case you are not aware of said API): 
For python, I like the [`pathlib.Path`](https://docs.python.org/3/library/pathlib.html#concrete-paths)-api. Much ncier than the older OS-api.

Would clean up this code a bit, but lets not do this in this PR